kubernetes/doc.go

```diff
@@ -0,0 +1,93 @@
+// Copyright © 2018 Genome Research Limited
+// Author: Theo Barber-Bany
+// <tb15@sanger.ac.uk>.
+//
+//  This file is part of wr.
+//
+//  wr is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  wr is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU Lesser General Public License for more details.
+//
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with wr. If not, see <http://www.gnu.org/licenses/>.
+
+package kubedoc
+
+/*
+This is intended as a high level overview of all kubernetes related code in the
+WR codebase and explanation of how it fits together. A general understanding of
+kubernetes concepts is assumed, along with an understanding of client-go
+(https://github.com/kubernetes/client-go).
+
+See the links in the deployment and scheduler controller docs for information on
+controllers.
+
+Below is a diagram showing the packages each step of the deployment and
+operation. It is also the order in which I would recommend reading each
```

Comment:
First sentence needs commas. Also, make the documentation less personal and conversational by rewording the second sentence something like "It is also the recommended order in which to read each package's documentation.'